tests/integration/utils/readers/test_scientific_reader.py

```
@@ -234,14 +313,121 @@ def test_netcdf_global_attributes_listed(self, tmp_path: Path) -> None:
         assert "Global Attributes" in result
         assert "title" in result
 
+    def test_netcdf_mocked_success_without_netcdf_dependency(self, tmp_path: Path) -> None:
+        from file_organizer.utils.readers import scientific as sci_module
+
+        path = tmp_path / "mocked.nc"
+        path.write_text("placeholder", encoding="utf-8")
+
+        class FakeDimension:
+            def __init__(self, size: int, unlimited: bool = False) -> None:
```

Ruff (0.15.10)

[warning] Boolean-typed positional argument in function definition

(FBT001)